project_task_name_with_id/tests/test_project_task_id.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import re
+
+from odoo.tests.common import TransactionCase
+
+
+class TestProjectTaskID(TransactionCase):
+    def setUp(self):
```

Comment:
Just a minor comment, test should be using @classsmethod
```suggestion
    @classmethod
    def setUpClass(cls):
```

Reply:
Done!